### ISSUE TITLE
Should be able to pass options through to map methods as you'd be able to do with a native mapbox implementation (e.g. fitBounds)

### ISSUE THREAD

**Issue by sdl60660** [beyonk-adventures/svelte-mapbox#30]
I noticed while using the fitBounds method, that it didn't support passing through the options available to that [mapbox function](https://docs.mapbox.com/mapbox-gl-js/api/map/#map#fitbounds), such as turning off animation (which is what I was trying to do).

It seems to me that this should be as straightforward as adding an optional parameter on your Map.svelte component's methods, which is what I did locally to make it work. For example:

```
export function fitBounds (bbox) {
  queue.send('fitBounds', [ bbox ])
}
```

Just becomes...

```
export function fitBounds (bbox, options={}) {
    queue.send('fitBounds', [ bbox, options ])
}
```

I'd be happy to submit a pull request to this effect, but I'm not sure if there was a reason things were implemented this way, without options.

**Comment by antony** [comment]
There isn't a specific way other than I didn't need the functionality :)

Feel free to open a PR to make this change!

**Comment by CraigChamberlain** [comment]
I think this held me back just now.  Is anyone working on a solution?  I'm happy to have a go if no-one else is.

**Comment by CraigChamberlain** [comment]
https://github.com/beyonk-adventures/svelte-mapbox/pull/39 ??

**Comment by sdl60660** [comment]
@CraigChamberlain Submitted it as a pull request. I ended up just changing it in the npm package to make an implementation work, but yeah it's a quick fix moving forward.

**Comment by sdl60660** [comment]
@CraigChamberlain I submitted it as a pull request.

**Comment by antony** [comment]
Released as v7.6.0 :) thanks!
